src/Coordination/SnapshotableHashTable.h

```diff
@@ -98,15 +98,10 @@ class SnapshotableHashTable
 
     enum OperationType
     {
-        INSERT = 0,
-        INSERT_OR_REPLACE = 1,
-        ERASE = 2,
-        UPDATE_VALUE = 3,
-        GET_VALUE = 4,
-        FIND = 5,
-        CONTAINS = 6,
-        CLEAR = 7,
-        CLEAR_OUTDATED_NODES = 8
+        INSERT_OR_REPLACE = 0,
```

Comment:
just to double check: this change does not introduce any backward incompatibility, right?

Reply:
yes it's stored nowhere, it is not sent anywhere, everything is local
I just cleaned it up a bit, half of the values were not even used anywhere...